src/box/space_cache.h

```diff
@@ -97,6 +97,13 @@ space_by_id(uint32_t id);
 struct space *
 space_by_name(const char *name);
 
+/**
+ * Find minimal available id, which is it lower than current_id.
+ * If there is no available id, BOX_SPACE_MAX + 1 is returned.
+ */
+uint32_t
+space_available_id(uint32_t current_id);
```

Comment:
Rather bad name - it's not really clear what the function does. Also, it's named like it works with a space object while in fact it doesn't. Better change it to `space_cache_find_next_unused_id(min_id)`.

Reply:
Done